include/circt/Conversion/CalyxNative.h

```diff
@@ -0,0 +1,32 @@
+//===- CalyxNative.h - Calyx Native pass ------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file declares passes which together will lower the Calyx dialect to the
+// HW dialect.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef CIRCT_CONVERSION_CALYXNATIVE_H
+#define CIRCT_CONVERSION_CALYXNATIVE_H
+
+#include "circt/Dialect/Calyx/CalyxOps.h"
+#include "circt/Support/LLVM.h"
+#include <memory>
+
+namespace mlir {
+class Pass;
+} // namespace mlir
+
+namespace circt {
+namespace calyx {} // namespace calyx
+
+std::unique_ptr<mlir::Pass> createCalyxNativePass();
```

Comment:
Wild guess: should this be in a `circt` namespace?

Reply:
so `circt::createCalyxNaitvePass`?

Reply:
`namespace circt { std::unique_ptr<mlir::Pass> createCalyxNativePass(); }`

Reply:
Actually, potentially need to have the `calyx` namespace contain the symbol. Trying that now!

Reply:
You define `circt::createCalyxNativePass` in your .cpp file, hence why I assumed that was the issue

Reply:
Well, in that case, it was already in a `circt` namespace (see line 25). I've tried committing your suggestion in case that changes anything so let's see

Reply:
Ah sorry about that, GitHub cut that out from my snapshot of the code. It seems https://github.com/llvm/circt/pull/5879/commits/1ff1e6b95fea0ae683e944ee5b847f808a839211 fixes it though! Nice catch, that has gotten me before as well. I wish it would produce a better error message.